internal/app/connectconformance/testsuites/data/grpc_web_client_trailers.yaml

```diff
@@ -122,18 +122,13 @@ testCases:
                   value: [ "*" ]
                 - name: content-type
                   value: [ "application/grpc-web" ]
-                - name: x-custom-header
-                  value: [ "foo" ]
                 - name: x-custom-trailer
                   value: [ "bing" ]
                 - name: grpc-status
                   value: [ "9" ]
                 - name: grpc-message
                   value: [ "error" ]
     expectedResponse:
-      responseHeaders:
-        - name: x-custom-header
-          value: [ "foo" ]
```

Comment:
🤦 this was just wrong. All of the metadata were in the same bucket in the trailers-only response, so the expected response should not have them split up. I just removed "headers" entirely from these since they don't make sense.